package.json

```diff
@@ -33,6 +33,7 @@
     "eslint-plugin-promise": "^4.0.1",
     "eslint-plugin-standard": "^4.0.0",
     "mousetrap": "^1.6.2",
+    "mousetrap-pause": "^1.0.0",
```

Comment:
Looks like this package is unused. Took a look at the source of that npm package and it is fancier than what we need. 

This version https://github.com/ccampbell/mousetrap/blob/master/plugins/pause/mousetrap-pause.js seems like the simplest approach, so we could just use that one by adding it to js/vendor instead of using npm.

Reply:
Tangentially, is this mousetrap-pause change related to the websocket json formatting fix?